packages/card-picker/tests/dummy/app/router.js

```diff
@@ -7,6 +7,7 @@ const Router = EmberRouter.extend({
 });
 
 Router.map(function() {
+  this.route('close-button');
```

Comment:
@jorgelainfiesta can we just get this changed to the application template please? Just so that when people spin up the dummy app for this it shows them something without needing to go to the `close-button` route

Reply:
Sure, I've moved everything to the application route.